tests/test_security_xss.py

```
@@ -0,0 +1,76 @@
+import pytest
+import html
+import asyncio
+from datetime import datetime, timezone
+from pathlib import Path
+from swarm.api.services.run_state import RunStateManager
+from swarm.api.routes.runs_control import _write_stop_report
+from swarm.api.routes.runs_models import StopReportInfo
+
+def test_stop_report_xss_prevention(tmp_path):
+    """Test that stop_report.md sanitizes user inputs preventing XSS."""
+    async def run_test():
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)